backend/src/main/java/fr/anettehaferkorn/backend/wineFinderAPI/model/FilterDTO.java

```diff
@@ -0,0 +1,13 @@
+package fr.anettehaferkorn.backend.wineFinderAPI.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class FilterDTO {
```

Comment:
This class could probably be replaced by a record (since Java 14). It would look like this:
```java
public record FilterDTO(String variety, String region) {
}
```